tools/postinstall

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eu
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+pod_install() {
+  [[ "$OSTYPE" == "darwin"* ]] ||
+    return 0
+  hash pod 2>/dev/null || {
+        cat >&2 <<EOF
+The postinstall script (see scripts.postinstall in package.json)
+requires CocoaPods. Please run
```

Comment:
Makes sense. I think it's nice to include something that identifies this output as concerning our own project's direct needs, rather than some random stuff you might see from `yarn` (though I also recognize the importance of minimizing `yarn`'s output where we can).

Saying "The Zulip mobile app" does this nicely.